nuxt.config.ts

```diff
@@ -59,24 +59,25 @@ export default defineNuxtConfig({
   i18n: {
     locales: [
       {
-        code: "en",
+        code: 'en',
```

Comment:
Well, I just reminded that ESLint might fail to fix the coding style because this repository still uses old ESLint v8.

If ESLint doesn't work properly, you don't have to fix the coding style. I will fix it later.

Reply:
I won’t be able to make these changes before 6 PM (UTC+8). 
If it’s urgent, feel free to modify them.

Reply:
No problem, it is not an urgent task.
I often post comments in the early morning but feel free to reply after school or any time convenient for you.

(I hear that Japanese teens are implicitly forced to reply to the text chat as soon as possible, but is it the same in China? I usually reply after several hours, so you don't have to try reply immediately either.)

Reply:
Maybe I’m one of the few exceptions allowed to use a phone at school

But obviously, I can’t bring my computer to school just for non-academic stuff lol